packages/pn-commons/src/components/CodeModal/__test__/CodeInput.test.tsx

```diff
@@ -105,7 +105,7 @@ describe('CodeInput Component', () => {
     act(() => (codeInputs[2] as HTMLInputElement).focus());
     await userEvent.keyboard('{Backspace}');
     await waitFor(() => {
-      expect(codeInputs[2]).toHaveValue('');
+      expect(codeInputs[2]).toBeEmptyDOMElement();
```

Comment:
the toBeEmptyDOMElement is to check if an element is empty (i.e. doesn't have any element inside). So an input always will be empty.
Restore the toHaveValue('')

Reply:
Okay about toBeEmptyDOMElement, btw using toHaveValue('') sometimes the test fails